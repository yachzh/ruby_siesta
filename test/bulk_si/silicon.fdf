SystemLabel   silicon
NumberOfAtoms   2
NumberOfSpecies   1
LatticeConstant   1.0 Ang
%block LatticeVectors
     0.00000000      2.71500000      2.71500000
     2.71500000      0.00000000      2.71500000
     2.71500000      2.71500000      0.00000000
%endblock LatticeVectors

AtomicCoordinatesFormat   Ang
%block ChemicalSpeciesLabel
  1   14   Si
%endblock ChemicalSpeciesLabel

%block AtomicCoordinatesAndAtomicSpecies
     0.00000000      0.00000000      0.00000000   1
     1.35750000      1.35750000      1.35750000   1
%endblock AtomicCoordinatesAndAtomicSpecies

PAO.BasisSize   DZP
PAO.EnergyShift   0.02 Ry
Mesh.Cutoff   300 Ry
WriteMullikenPop   0
SCF.Mixer.Weight   0.25
SCF.Mixer.History   2
SCF.DM.Tolerance   0.0001
SCF.H.Converge   true
SCF.H.Tolerance   1.0 meV
MaxSCFIterations   1000
OccupationFunction   FD
OccupationMPOrder   1
ElectronicTemperature   300 K
XC.Functional   LDA
XC.Authors   PW92

%block kgrid_Monkhorst_Pack
     9    0    0   0.0
     0    9    0   0.0
     0    0    9   0.0
%endblock kgrid_Monkhorst_Pack

