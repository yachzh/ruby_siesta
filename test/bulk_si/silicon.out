Siesta Version  : 4.1.5
Architecture    : x86_64
Compiler version: GNU Fortran (GCC) 13.2.1 20240316 (Red Hat 13.2.1-7)
Compiler flags  : gfortran -O2 -fPIC -ftree-vectorize -fallow-argument-mismatch
PP flags        : -DFC_HAVE_ABORT -DCDF -DNCDF -DNCDF_4 -DSIESTA__DIAG_2STAGE -DSIESTA__MRRR
Libraries       : libsiestaLAPACK.a libsiestaBLAS.a libncdf.a libfdict.a -lnetcdff -lnetcdf -lhhdf5_hl -lhdf5 -lz
SERIAL version
NetCDF support
NetCDF-4 support

* Running in serial mode
>> Start of run:  20-AUG-2024  10:03:23

                           ***********************       
                           *  WELCOME TO SIESTA  *       
                           ***********************       

reinit: Reading from silicon.fdf

reinit: -----------------------------------------------------------------------
reinit: System Name: 
reinit: -----------------------------------------------------------------------
reinit: System Label: silicon
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   14 Label: Si

Ground state valence configuration:   3s02  3p02
Reading pseudopotential information in formatted form from Si.psf

Valence configuration for pseudopotential generation:
3s( 2.00) rc: 1.77
3p( 2.00) rc: 1.96
3d( 0.00) rc: 2.11
4f( 0.00) rc: 2.11
For Si, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Si                   Z=  14    Mass=  28.090        Charge= 0.17977+309
Lmxo=1 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=3
          i=1  nzeta=2  polorb=0  (3s)
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=3
          i=1  nzeta=2  polorb=1  (3p)
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for Si                    (Z =  14)

read_vps: Pseudopotential generation method:
read_vps: ATM 3.2.2 Troullier-Martins                       
Valence charge for ps generation:    4.00000

xc_check: Exchange-correlation functional:
xc_check: Perdew-Wang 1992
xc_check: WARNING: Pseudopotential generated with LDA CA functional
V l=0 = -2*Zval/r beyond r=  1.7304
V l=1 = -2*Zval/r beyond r=  1.9124
V l=2 = -2*Zval/r beyond r=  2.0613
V l=3 = -2*Zval/r beyond r=  2.0613
All V_l potentials equal beyond r=  2.0613
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  2.0613

VLOCAL1: 99.0% of the norm of Vloc inside     10.216 Ry
VLOCAL1: 99.9% of the norm of Vloc inside     23.282 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    2.45560
atom: Maximum radius for r*vlocal+2*Zval:    2.14012
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2
GHOST: No ghost state for L =  3

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  2.278153   el= -0.798543   Ekb=  4.869008   kbcos=  0.287219
   l= 1   rc=  2.306811   el= -0.305674   Ekb=  1.488641   kbcos=  0.299482
   l= 2   rc=  2.335829   el=  0.002136   Ekb= -2.242464   kbcos= -0.060154
   l= 3   rc=  2.425092   el=  0.003224   Ekb= -0.691507   kbcos= -0.006809

KBgen: Total number of Kleinman-Bylander projectors:   16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 3s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.020000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    5.007352
                 energy =   -0.779804
                kinetic =    0.585544
    potential(screened) =   -1.365349
       potential(ionic) =   -3.844139

   izeta = 2
                 rmatch =    4.418952
              splitnorm =    0.150000
                 energy =   -0.680710
                kinetic =    0.898456
    potential(screened) =   -1.579166
       potential(ionic) =   -4.162544

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 3p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.020000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    6.270866
                 energy =   -0.287480
                kinetic =    0.877699
    potential(screened) =   -1.165179
       potential(ionic) =   -3.426628

   izeta = 2
                 rmatch =    5.007352
              splitnorm =    0.150000
                 energy =   -0.192533
                kinetic =    1.274192
    potential(screened) =   -1.466725
       potential(ionic) =   -3.923077

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 3p

   izeta = 1
                     rc =    6.270866
                 energy =    0.432971
                kinetic =    1.296204
    potential(screened) =   -0.863233
       potential(ionic) =   -2.929353
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 3s( 2.00)                                                            
 3p( 2.00)                                                            
Vna: chval, zval:    4.00000   4.00000

Vna:  Cut-off radius for the neutral-atom potential:   6.270866

atom: _________________________________________________________________________

prinput: Basis input ----------------------------------------------------------

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   14 Si                      # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
Si                    2                    # Species label, number of l-shells
 n=3   0   2                         # n, l, Nzeta 
   5.007      4.419   
   1.000      1.000   
 n=3   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   6.271      5.007   
   1.000      1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file Si.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      2.56530   2.56530   2.56530  1        2

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      2    26    32

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   300.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9978 K
redata: Fix the spin of the system                  =   F
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:     8.000000
Total ionic charge:     8.000000
 Kpoints in:          405 . Kpoints trimmed:          365

siesta: k-grid: Number of k-points =   365
siesta: k-grid: Cutoff (effective) =    17.278 Ang
siesta: k-grid: Supercell and displacements
siesta: k-grid:    9   0   0      0.000
siesta: k-grid:    0   9   0      0.000
siesta: k-grid:    0   0   9      0.000

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000

superc: Internal auxiliary supercell:     5 x     5 x     5  =     125
superc: Number of atoms, orbitals, and projectors:    250   3250   4000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     5 x     5 x     5  =     125
superc: Number of atoms, orbitals, and projectors:    250   3250   4000

outcell: Unit cell vectors (Ang):
        0.000000    2.715000    2.715000
        2.715000    0.000000    2.715000
        2.715000    2.715000    0.000000

outcell: Cell vector modules (Ang)   :    3.839590    3.839590    3.839590
outcell: Cell angles (23,13,12) (deg):     60.0000     60.0000     60.0000
outcell: Cell volume (Ang**3)        :     40.0258
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=26 nrows=26 sparsity=58.9024 nnzs=39818, refcount: 7>
  <dData1D:(new from dSpData1D) n=39818, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=26 nrows=26 sparsity=58.9024 nnzs=39818, refcount: 8>
  <dData2D:DM n=39818 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      38     305
New grid distribution:   1
           1       1:   18    1:   18    1:   18

InitMesh: MESH =    36 x    36 x    36 =       46656
InitMesh: (bp) =    18 x    18 x    18 =        5832
InitMesh: Mesh cutoff (required, used) =   300.000   364.442 Ry
ExtMesh (bp) on 0 =    98 x    98 x    98 =      941192
PhiOnMesh: Number of (b)points on node 0 =                 5832
PhiOnMesh: nlist on node 0 =               498794

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -75.089324
siesta: Eions   =       377.610449
siesta: Ena     =       112.762046
siesta: Ekin    =        82.398449
siesta: Enl     =        28.424249
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         3.359960
siesta: DUscf   =         0.287240
siesta: DUext   =         0.000000
siesta: Exc     =       -65.171663
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -215.959582
siesta: Etot    =      -215.550168
siesta: FreeEng =      -215.550168

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -215.959582     -215.550168     -215.550168  1.822097 -4.117520  0.200243
timer: Routine,Calls,Time,% = IterSCF        1       1.486  49.05
   scf:    2     -215.554371     -215.552311     -215.552311  0.004516 -4.083397  0.136520
   scf:    3     -215.555695     -215.554206     -215.554207  0.009623 -4.007894  0.002942
   scf:    4     -215.554211     -215.554209     -215.554209  0.000182 -4.007549  0.002228
   scf:    5     -215.554214     -215.554212     -215.554212  0.000505 -4.006500  0.000273
   scf:    6     -215.554212     -215.554212     -215.554212  0.000003 -4.006433  0.000206

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000029842
max |H_out - H_in|      (eV) :     0.0002058908
SCF cycle converged after 6 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      38     305

siesta: E_KS(eV) =             -215.5542

siesta: E_KS - E_eggbox =      -215.5542

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000000   -0.000000    0.000000
----------------------------------------
   Max    0.000000
   Res    0.000000    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.000000    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):        9.39        9.39        9.39        0.00        0.00        0.00
(Free)E + p*V (eV/cell)     -215.7887
Target enthalpy (eV/cell)     -215.5542

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -75.353050
siesta: Eions   =       377.610449
siesta: Ena     =       112.762046
siesta: Ekin    =        81.990842
siesta: Enl     =        28.369448
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         3.725260
siesta: DUscf   =         0.265604
siesta: DUext   =         0.000000
siesta: Exc     =       -65.056963
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -215.554212
siesta: Etot    =      -215.554212
siesta: FreeEng =      -215.554212

siesta: Final energy (eV):
siesta:  Band Struct. =     -75.353050
siesta:       Kinetic =      81.990842
siesta:       Hartree =      14.797040
siesta:       Edftu   =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:   Exch.-corr. =     -65.056963
siesta:  Ion-electron =    -102.075917
siesta:       Ion-ion =    -145.209214
siesta:       Ekinion =       0.000000
siesta:         Total =    -215.554212
siesta:         Fermi =      -4.006433

siesta: Stress tensor (static) (eV/Ang**3):
siesta:     0.005859    0.000000    0.000000
siesta:     0.000000    0.005859    0.000000
siesta:     0.000000    0.000000    0.005859

siesta: Cell volume =         40.025752 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:          -0.00006381         -0.00006381  Ry/Bohr**3
siesta:          -0.00585884         -0.00585884  eV/Ang**3
siesta:          -9.38699866         -9.38699867  kBar
(Free)E+ p_basis*V_orbitals  =        -214.267093
(Free)Eharris+ p_basis*V_orbitals  =        -214.267093

cite: Please see "silicon.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 4.1.5
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =      10.404
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1      10.359      10.359   100.00
Setup                  1       0.124       0.124     1.20
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1      10.227      10.227    98.73
state_init             1       0.130       0.130     1.26
hsparse                1       0.007       0.007     0.07
overlap                1       0.109       0.109     1.05
Setup_H0               1       1.289       1.289    12.44
naefs                  2       0.000       0.000     0.00
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.055       0.111     1.07
nlefsm                 2       0.382       0.765     7.38
DHSCF_Init             1       0.645       0.645     6.22
DHSCF1                 1       0.041       0.041     0.40
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.604       0.604     5.83
REMESH                 1       0.114       0.114     1.10
REORD                 24       0.000       0.002     0.02
PHION                  1       0.448       0.448     4.33
COMM_BSC              35       0.000       0.002     0.02
POISON                 9       0.002       0.017     0.16
fft                   18       0.001       0.011     0.11
IterSCF                6       1.250       7.500    72.40
setup_H                7       0.282       1.977    19.08
DHSCF                  8       0.380       3.041    29.36
DHSCF3                 8       0.282       2.257    21.79
rhoofd                 8       0.156       1.248    12.04
XC                     8       0.017       0.135     1.30
GXC-CellXC             8       0.017       0.134     1.29
cellXC                 8       0.017       0.134     1.29
vmat                   8       0.107       0.854     8.24
compute_dm             6       0.920       5.519    53.27
diagon                 6       0.920       5.518    53.27
c-eigval            2190       0.001       1.964    18.96
c-buildHS           2190       0.001       1.795    17.33
cdiag               4380       0.000       0.526     5.08
cdiag1              4380       0.000       0.036     0.35
cdiag2              4380       0.000       0.076     0.73
cdiag3              4380       0.000       0.386     3.72
c-eigvec            2190       0.001       2.164    20.89
cdiag4              2190       0.000       0.009     0.09
c-buildD            2190       0.001       1.377    13.29
MIXER                  5       0.000       0.002     0.02
PostSCF                1       1.306       1.306    12.61
DHSCF4                 1       0.784       0.784     7.57
dfscf                  1       0.741       0.741     7.15
overfsm                1       0.008       0.008     0.08
state_analysis         1       0.000       0.000     0.00
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.007       0.007     0.07
siesta_analysis        1       0.007       0.007     0.07
optical                1       0.000       0.000     0.00
  
>> End of run:  20-AUG-2024  10:03:33
Job completed
